generator.ts

```diff
@@ -163,7 +165,7 @@ export async function generate(token: string, configYaml: string): Promise<Repor
     set.add(issues)
 
     heading(`Processing target: '${processor.target}' with processor: '${processor.name}'`)
-    processingModule.process(target, config, set)
+    await processingModule.process(target, config, set, github)
```

Comment:
processing sections need to write so we hand over a github client already setup with the PAT and a cache plugin all setup.